wrappers/Julia/CoolProp.jl

```diff
@@ -55,6 +55,7 @@ end
 
 # CoolProp::get_parameter_information_string
 function get_parameter_information_string(Key::AbstractString)
+message_buffer[1:4] = UInt8[108 111 110 103] # "long" written in UInt8
```

Comment:
@mikekaganski I first thought too, but the result is a `ASCIIString`. I also tried `utf8()`, but then the result is `UTF8String`.
So I can't insert them in the buffer as it requires `Array{UInt8,1}`.

Theses strings could be send to the CoolProp library, but the problem is that I don't have the buffer anymore to get the result back.